2.0.0-alpha.5

matteobruni opened the pull request.

matteobruni committed:
build: fix jquery component rollup config

matteobruni committed:
Merge branch 'dev' of https://github.com/matteobruni/tsparticles into dev

matteobruni committed:
build: angular component publish fix

matteobruni committed:
build: renamed tsparticles.core to tsparticles.engine

matteobruni committed:
build: added downlevel-dts configuration to all vanilla packages

ghost commented:
## DeepCode's analysis on [#abc780](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/fc63cfd1f13b04cadf08a7d81524fa12a475de2f/matteobruni/tsparticles/abc780a8eb7c71322251a70d0cfd687fecb91925/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) found:

- :information_source: **1** minor issue. :point_down:
## Top issues

<table>
  <thead>
    <tr>
      <th align="left">Description</th>
      <th align="left">Example fixes</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <td width="77%">Use "utf8", which Node uses, instead of "utf-8" (in fs.writeFileSync). Occurrences: <ul><li><a href="https://github.com/matteobruni/tsparticles/blob/abc780a8eb7c71322251a70d0cfd687fecb91925/presets/bubbles/scripts/distfiles.js#L17">distfiles.js:17</a></li></ul></td>
      <td width="23%"><a href="https://www.deepcode.ai/app/gh/matteobruni/tsparticles/fc63cfd1f13b04cadf08a7d81524fa12a475de2f/matteobruni/tsparticles/abc780a8eb7c71322251a70d0cfd687fecb91925/pr/_/%2Fpresets%2Fbubbles%2Fscripts%2Fdistfiles.js/javascript%2Fdc_interfile_project%2FUtf8Literal/code/?utm_source=gh_review&">:wrench: Example fixes</a></td>
    </tr>
  </tbody>
</table>


#### 👉 View analysis in [**DeepCode’s Dashboard**](https://www.deepcode.ai/app/gh/matteobruni/tsparticles/fc63cfd1f13b04cadf08a7d81524fa12a475de2f/matteobruni/tsparticles/abc780a8eb7c71322251a70d0cfd687fecb91925/pr/_/%2F/code/?utm_source=gh_review&c=0&w=0&i=1&) | [_Configure the bot_](https://www.deepcode.ai/app/gh/?ownerconfig=matteobruni)

matteobruni committed:
build: actual options in container, preparing for responsive options

matteobruni committed:
feat: responsive options
feat: resize keeps particles in place

matteobruni committed:
feat: added overlap options to collisions
feat: added new bubbles preset (emitterAngled in demo)

matteobruni committed:
build: fixed editor

matteobruni committed:
build: update bubbles preset with prettier/eslint

matteobruni committed:
build: renamed options and fullOptions to actualOptions and options
build: fixed jquery component build

matteobruni committed:
build: responsive demo added

matteobruni committed:
chore(release): published new version

 - ng-particles@2.3.0
 - inferno-particles@1.3.0
 - jquery-particles@1.20.0
 - preact-particles@1.20.0
 - react-tsparticles@1.20.0
 - svelte-particles@1.3.0
 - particles.vue@2.3.0
 - particles.vue3@1.3.0
 - tsparticles-editor@1.20.0
 - tsparticles@1.20.0
 - ng-particles-demo@1.5.0
 - editor-demo@1.5.0
 - inferno-particles-demo@1.3.0
 - jquery-particles-demo@1.5.0
 - tsparticles-demo@1.5.0
 - preact-particles-demo@1.5.0
 - react-tsparticles-demo@1.5.0
 - svelte-demo@1.3.0
 - particles.vue-demo@1.5.0
 - particles.vue3-demo@1.3.0
 - tsparticles-preset-60fps@1.5.0
 - tsparticles-preset-background-mask@1.5.0
 - tsparticles-preset-basic@1.5.0
 - tsparticles-preset-big-circles@1.5.0
 - tsparticles-preset-bouncing@1.5.0
 - tsparticles-preset-bubbles@1.5.0
 - tsparticles-preset-fire@1.5.0
 - tsparticles-preset-font-awesome@1.5.0
 - tsparticles-preset-fountain@1.5.0
 - tsparticles-preset-snow@1.5.0
 - tsparticles-preset-stars@1.5.0
 - tsparticles-shape-bubble@1.5.0
 - tsparticles-shape-heart@1.5.0
 - tsparticles-shape-multiline-text@1.5.0
 - tsparticles-shape-spiral@1.5.0
 - cra-template-particles-typescript@1.3.0
 - cra-template-particles@1.3.0
 - tsparticles-website@1.20.0

matteobruni committed:
Merge pull request #1375 from matteobruni/v1

1.20.0

matteobruni committed:
Merge branch 'main' into dev

matteobruni committed:
build: merge fix

matteobruni committed:
build: fixed all presets

matteobruni committed:
chore(release): published new version

 - tsparticles@2.0.0-alpha.5
 - tsparticles-slim@2.0.0-alpha.5
 - ng-particles@3.0.0-alpha.5
 - inferno-particles@2.0.0-alpha.5
 - jquery-particles@2.0.0-alpha.5
 - preact-particles@2.0.0-alpha.5
 - react-tsparticles@2.0.0-alpha.5
 - svelte-particles@2.0.0-alpha.5
 - particles.vue@3.0.0-alpha.5
 - particles.vue3@2.0.0-alpha.5
 - ng-particles-demo@2.0.0-alpha.5
 - editor-demo@2.0.0-alpha.5
 - inferno-particles-demo@2.0.0-alpha.5
 - jquery-particles-demo@2.0.0-alpha.5
 - preact-particles-demo@2.0.0-alpha.5
 - react-tsparticles-demo@2.0.0-alpha.5
 - svelte-demo@2.0.0-alpha.5
 - tsparticles-demo@2.0.0-alpha.5
 - particles.vue-demo@2.0.0-alpha.5
 - particles.vue3-demo@2.0.0-alpha.5
 - tsparticles-editor@2.0.0-alpha.5
 - tsparticles-engine@2.0.0-alpha.5
 - tsparticles-interaction-light@2.0.0-alpha.5
 - tsparticles-interaction-external-attract@2.0.0-alpha.5
 - tsparticles-interaction-external-bounce@2.0.0-alpha.5
 - tsparticles-interaction-external-bubble@2.0.0-alpha.5
 - tsparticles-interaction-external-connect@2.0.0-alpha.5
 - tsparticles-interaction-external-grab@2.0.0-alpha.5
 - tsparticles-interaction-external-repulse@2.0.0-alpha.5
 - tsparticles-interaction-external-trail@2.0.0-alpha.5
 - tsparticles-interaction-particles-attract@2.0.0-alpha.5
 - tsparticles-interaction-particles-collisions@2.0.0-alpha.5
 - tsparticles-interaction-particles-links@2.0.0-alpha.5
 - tsparticles-interaction-particles-move@2.0.0-alpha.5
 - tsparticles-interaction-particles-parallax@2.0.0-alpha.5
 - tsparticles-interaction-particles-repulse@2.0.0-alpha.5
 - tsparticles-plugin-absorbers@2.0.0-alpha.5
 - tsparticles-plugin-emitters@2.0.0-alpha.5
 - tsparticles-plugin-infection@2.0.0-alpha.5
 - tsparticles-plugin-polygon-mask@2.0.0-alpha.5
 - tsparticles-preset-big-circles@2.0.0-alpha.5
 - tsparticles-preset-bubbles@2.0.0-alpha.5
 - tsparticles-preset-fire@2.0.0-alpha.5
 - tsparticles-preset-fountain@2.0.0-alpha.5
 - tsparticles-preset-links@2.0.0-alpha.5
 - tsparticles-preset-sea-anemone@2.0.0-alpha.5
 - tsparticles-preset-snow@2.0.0-alpha.5
 - tsparticles-preset-stars@2.0.0-alpha.5
 - tsparticles-shape-bubble@2.0.0-alpha.5
 - tsparticles-shape-circle@2.0.0-alpha.5
 - tsparticles-shape-heart@2.0.0-alpha.5
 - tsparticles-shape-image@2.0.0-alpha.5
 - tsparticles-shape-line@2.0.0-alpha.5
 - tsparticles-shape-multiline-text@2.0.0-alpha.5
 - tsparticles-shape-polygon@2.0.0-alpha.5
 - tsparticles-shape-spiral@2.0.0-alpha.5
 - tsparticles-shape-square@2.0.0-alpha.5
 - tsparticles-shape-star@2.0.0-alpha.5
 - tsparticles-shape-text@2.0.0-alpha.5
 - cra-template-particles-typescript@2.0.0-alpha.5
 - cra-template-particles@2.0.0-alpha.5
 - tsparticles-updater-angle@2.0.0-alpha.5
 - tsparticles-updater-color@2.0.0-alpha.5
 - tsparticles-updater-life@2.0.0-alpha.5
 - tsparticles-updater-opacity@2.0.0-alpha.5
 - tsparticles-updater-orbit@2.0.0-alpha.5
 - tsparticles-updater-out-modes@2.0.0-alpha.5
 - tsparticles-updater-size@2.0.0-alpha.5
 - tsparticles-updater-stroke-color@2.0.0-alpha.5

matteobruni committed:
build: package.json dist updated
